iphone/Classes/TiCalendarAttendee.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#import "TiProxy.h"
+
+#ifdef USE_TI_CALENDAR
+
+@class CalendarModule;
+@class EKParticipant;
+
+@interface TiCalendarAttendee : TiProxy {
+@private
+    CalendarModule* module;
+    EKParticipant* participant;
+    BOOL isOrganiser;
+}
+
+-(id)_initWithPageContext:(id<TiEvaluator>)context participant:(EKParticipant*)participant_ isOrganiser:(BOOL)_isOrganiser module:(CalendarModule*)module_;
```

Comment:
This class is only used as a data object when we are fetching the attendees, there is no need to make them separate setters. Also we use this same method structure in multiple places all of which inherit from TiProxy.  We actually use it when initializing TiCalandarEvent, so in theory should not cause any crashes.